helm-service/deploy/service.yaml

```diff
@@ -42,8 +42,6 @@ spec:
           value: 'http://configuration-service:8080'
```

Comment:
I think now that skaffold uses `helm` to deploy, we can complete remove this manifest.